STCommon/src/commonMain/kotlin/com/infomaniak/multiplatform_swisstransfer/common/models/TransferType.kt

```diff
@@ -20,6 +20,4 @@ package com.infomaniak.multiplatform_swisstransfer.common.models
 enum class TransferType(val matomoValue: String) {
     LINK("link"),
     MAIL("mail"),
-    QR_CODE("qrCode"),
-    PROXIMITY("proximity"),
```

Comment:
We should also report these changes to the swisstransfer matomo file that lists all of the available matomo events